models/provisioning-schemas.ts

```diff
@@ -0,0 +1,141 @@
+// temporary schema to use for validating /provision-job request
+import { ProvisionRequestType } from "./provisioning-jobs";
+
+export const Clin = {
+  type: "object",
+  properties: {
+    number: { type: "string" },
+    popStartDate: { type: "string" },
+    popEndDate: { type: "string" },
+  },
+  required: ["number", "popStartDate", "popEndDate"],
+};
+
+export const TaskOrder = {
+  type: "object",
+  properties: {
+    number: { type: "string" },
+    // clins: {
+    //   type: "array",
+    //   items: { Clin },
+    // },
+  },
+  required: ["number", "clins"],
+};
+
+export const Administrator = {
+  type: "object",
+  properties: {
+    email: { type: "string" },
+    dodId: { type: "string" },
+    needsReset: { type: "boolean", default: false },
+  },
+  required: ["email", "dodId"],
+};
+
+export const AddPortfolioPayload = {
+  type: "object",
+  properties: {
+    name: { type: "string" },
+    // taskOrders: {
+    //   type: "array",
+    //   items: { $ref: TaskOrder },
+    // },
```

Comment:
Oh okay, just wanted to double check since it was a good amount. 

Yeah, it is tricky with `middy-ajv`. I wasn't sure if you figured something else out with the `$ref`s.